src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -4777,14 +4856,10 @@ class C
                             Row(4, TableIndex.MethodDef, EditAndContinueOperation.Default),
                             Row(2, TableIndex.TypeDef, EditAndContinueOperation.AddMethod),
                             Row(5, TableIndex.MethodDef, EditAndContinueOperation.Default),
-                            Row(1, TableIndex.Param, EditAndContinueOperation.Default),
-                            Row(2, TableIndex.Param, EditAndContinueOperation.Default),
                             Row(4, TableIndex.MethodDef, EditAndContinueOperation.AddParameter),
                             Row(3, TableIndex.Param, EditAndContinueOperation.Default),
                             Row(5, TableIndex.MethodDef, EditAndContinueOperation.AddParameter),
                             Row(4, TableIndex.Param, EditAndContinueOperation.Default),
-                            Row(1, TableIndex.CustomAttribute, EditAndContinueOperation.Default),
-                            Row(7, TableIndex.CustomAttribute, EditAndContinueOperation.Default),
                             Row(8, TableIndex.CustomAttribute, EditAndContinueOperation.Default),
```

Comment:
One of the quirks of CustomAttributes is that we emit all of them so that they can be deleted. ie, if the baseline has two methods, each with two attributes, and then gen 1 deletes one of the attributes on the first method, we'll emit all 4 attributes, changing the 2nd one to point to the 2nd method, and the last one to point nowhere.

This probably isn't covered by existing tests, but not emitting attributes for deleted members makes me worried about how these two features will interact. If we skip emitting a custom attribute for a deleted member, and also update an existing attribute to point to a different method, we might end up missing some data.

Reply:
Seems like something is wrong with the custom attributes even without this change. Will follow up with a separate PR.